PythonToHash

```diff
@@ -0,0 +1,27 @@
+# Python rogram to find the SHA-1 message digest of a file
+
+# importing the hashlib module
+import hashlib
```

Comment:
If your importing modules always remember to keep a requirments.txt listing the modules imported in the same folder as the program itself.